### PR TITLE
bump to v1.25

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ build:
   number: 0
   skip: true  # [not py36 or win]
   script: 
-   - python setup.py install --single-version-externally-managed --record record.txt
+   - script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (dbast) [inline recipe/meta.yaml]: script: script: "{{..."  seems to be one script too much. ":" is interpreted as dict, thats why travis is telling "TypeError: sequence item 0: expected str instance, dict found".

**Review by dbast — COMMENTED** [review]
see inline

File: recipe/meta.yaml (hunk 1 above)
Comment (basnijholt) [reply recipe/meta.yaml]: That's it! Thanks a lot 👍

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Pull request merged**